config/module_input.go

```diff
@@ -18,6 +18,13 @@ type ModuleInputConfig interface {
 	MonitorConfig
 }
 
+// ModuleInputConfigs is a collection of ModuleInputConfig
+type ModuleInputConfigs []ModuleInputConfig
+
+func DefaultModuleInputConfigs() *ModuleInputConfigs {
```

Comment:
Just wanted to point out that this currently isn't used, although I think you have plans for it in the future. Just wanted to make a note of it.

Reply:
As of this PR, DefaultModuleInputConfigs() is still currently being used outside of the config package e.g. controller, api, driver. I'll remove it in a subsequent PR when it's no longer being used.